controllers/userController.js

```diff
@@ -57,4 +57,64 @@ const emailCheck = async (req, res) => {
   }
 };
 
-module.exports = { signUp, signIn, emailCheck };
+const myPageShow = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.mypage(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const myAddress = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.myAddressShow(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const addressPatch = async (req, res) => {
+  try {
+    await userService.myAddressPatch(req);
+    return res.status(200).json({ message: "addressPatch Success" });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+const myInfo = async (req, res) => {
```

Comment:
```suggestion
const getUserProfile = async (req, res) => {
```
routes에서 리뷰드린 내용에 따라 해당 Method가 profile이 되는게 dao에서 가져오는 자원과 비교했을 때 더 명확할 것 같습니다.

Reply:
네 개인정보니까 프로필이 맞는거같아 수정하였습니다 !